starlingcaptureapi/iscn.py

```diff
@@ -0,0 +1,32 @@
+from . import config
+
+import logging
+import requests
+
+_logger = logging.getLogger(__name__)
+_REGISTER = f"{config.ISCN_SERVER}/iscn/new/"
+
+
+class Iscn:
+    """Handles interactions with ISCN"""
+
+    @staticmethod
+    def register(registration: dict) -> bool:
+        """Registers an asset on ISCN with the provided metadata.
+
+        Args:
+            registration: the complete contents of the ISCN registration; must comply with the
+                ISCN schema (https://github.com/likecoin/iscn-specs/tree/master/schema)
+
+        Returns:
+            True if the registration succeeded; False otherwise
+        """
+        response = requests.post(_REGISTER, json={"metadata": registration})
+
+        if not response.ok:
```

Comment:
I wonder how we may log failures in a way that can support a retry policy in the future. Same goes for Numbers and Open Timestamp registrations.